qt/scientific_interfaces/Indirect/IndirectCorrections.cpp

```diff
@@ -72,6 +72,7 @@ void IndirectCorrections::initLayout() {
             SLOT(showMessageBox(const QString &)));
   }
 
+  m_uiForm.pbSettings->setIcon(IndirectSettings::icon());
```

Comment:
Thanks for doing this, this was on my list! Could you add the same line to `IndirectDiffractionReduction.cpp`?